lib/commonmarker/renderer/html_renderer.rb

```diff
@@ -47,35 +47,43 @@ def list(node)
       @in_tight = old_in_tight
     end
 
-    def list_item(_)
+    def list_item(node)
       block do
-        container('<li>', '</li>') do
+        container("<li#{sourcepos(node)}>", '</li>') do
           out(:children)
         end
       end
     end
 
-    def blockquote(_)
+    def blockquote(node)
       block do
-        container("<blockquote>\n", '</blockquote>') do
+        container("<blockquote#{sourcepos(node)}>\n", '</blockquote>') do
           out(:children)
         end
       end
     end
 
-    def hrule(_)
+    def hrule(node)
       block do
-        out('<hr />')
+        out("<hr#{sourcepos(node)} />")
       end
     end
 
     def code_block(node)
       block do
-        out('<pre><code')
-        if node.fence_info && !node.fence_info.empty?
-          out(' class="language-', node.fence_info.split(/\s+/)[0], '">')
+        if @opts & CommonMarker::Config::Render::GITHUB_PRE_LANG != 0
```

Comment:
`const_get` gives me the heeby-jeebies but `ruby-enum` defines a `value` method which does the same thing.